packages/charts/react-charting/src/components/CommonComponents/CartesianChart.types.ts

```diff
@@ -81,6 +81,11 @@ export interface ICartesianChartStyleProps {
    * Prop to disable shrinking of the chart beyond a certain limit and enable scrolling when the chart overflows
    */
   enableReflow?: boolean;
+
+  /**
+   * opacity
```

Comment:
Add comment that this prop should only be used if absolutely necessary as it can impact the accessibility of the output chart color.